template/manifests/Service/draft.yaml

```diff
@@ -0,0 +1,30 @@
+templateName: "Service"
+description: "This template is used to create a generic Service for an application"
+type: "manifest"
+variables:
+  - name: "PORT"
+    type: "int"
+    kind: "port"
+    description: "the port exposed in the application"
+    default:
+      value: 80
+  - name: "APPNAME"
+    type: "string"
+    kind: "kubernetesResourceName"
+    description: "the name of the application"
+  - name: "PARTOF"
+    type: "string"
+    kind: "label"
+    description: "the label to identify which project the resource belong to"
+  - name: "GENERATORLABEL"
+    type: "string"
+    kind: "label"
+    description: "the label to identify who generated the resource"
+    default:
+      value: "draft"
+  - name: "TARGETPORT"
+    type: "int"
+    kind: "port"
+    description: "the port the Service uses to make the application accessible from outside the cluster"
```

Comment:
TARGETPORT and PORT descriptions should swap. The target port is the port the application exposes while the port is the one on the service